layers/+lang/emacs-lisp/packages.el

```diff
@@ -46,6 +47,19 @@
 (defun emacs-lisp/post-init-eldoc ()
   (add-hook 'emacs-lisp-mode-hook 'eldoc-mode))
 
+(defun emacs-lisp/init-auto-compile ()
+  (use-package auto-compile
+    :defer t
+    :init
+    (progn
+      (setq auto-compile-display-buffer nil
+            auto-compile-use-mode-line t
+            auto-compile-mode-line-counter t)
```

Comment:
I see.  I've been using `:config` in my old custom configuration because it plays better with the byte compiler checking for undefined variables, but you have a pretty strong point for `:init`.  I was really just being curious :relaxed: